cfgov/unprocessed/css/pages/external-site.less

```diff
@@ -42,4 +43,5 @@
 
 .external-site_reload-duration {
     .h2();
+    line-height: @base-font-size-px;
```

Comment:
Updated to `0` in https://github.com/cfpb/cfgov-refresh/commit/933a6d8738379a5546fa6f1573f1e53094ad4fe8